assets/img/recommendations/tom-goodman.webp:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]

```

Comment:
What's this file? Something leftover from WSL?

Reply:
Ah, a weird file from a .webp file I added. Will delete.